util/httputil/context.go

```diff
@@ -33,20 +33,24 @@ func ContextWithPath(ctx context.Context, path string) context.Context {
 
 // ContextFromRequest returns a new context from a requests with identifiers of
```

Comment:
LGTM. Can you just fix this comment?

```suggestion
// ContextFromRequest returns a new context from a request with identifiers of
```

Reply:
what about?

```suggestion
// ContextFromRequest returns a new context with identifiers of 
// the request to be used later when logging the query.
```